HttpRequest.cpp

```diff
@@ -1,120 +0,0 @@
-#include "HttpRequest.h"
-#include <vector>

```

Comment:
Would comment this stuff out instead of just deleting it. might be useful later?